assembly/pom.xml

```diff
@@ -193,19 +205,31 @@
       <dependencies>
         <dependency>
           <groupId>org.apache.spark</groupId>
-          <artifactId>spark-hive_${scala.binary.version}</artifactId>
+          <artifactId>xsql-hive_${scala.binary.version}</artifactId>
           <version>${project.version}</version>
         </dependency>
+        <dependency>
```

Comment:
As spark-hive don't have transitive dependency by other modules, there is no need to mark spark-hive has provided.

Reply:
OK. I understand now.
spark-mllib have the dependencies contains spark-catalyst and spark-sql, but spark-hive.